### DISCUSSION TITLE
wire:loading bound to dirty state?

### DISCUSSION THREAD

**Discussion by smares** [livewire/livewire#3497] [7. Bugs (archived)]
Given the following example: https://laravelplayground.com/#/snippets/25ff2e88-dde1-4751-aa36-61cfba388282

I would expect the `readonly` attribute to be set to the input whenever I click on the button, but in fact, it's only set when the state is dirty. So if you change the content of the input to something else and then click the button, `readonly` is set during the loading state. If you then press the button again without changing the text, `readonly` is not set. Is this a bug?

**Comment by joshhanley** [comment]
@smares hmm yeah I'd expect the readonly to apply whenever loading is happening. So it's probably a bug.

I've changed the category on this discussion to bug.
